test/remote/account_login_tests.js

```diff
@@ -184,6 +186,74 @@ describe('remote account login', () => {
     }
   )
 
+  describe('can force verificationMethod', () => {
+    let client, email
+    const password = 'foo'
+    before(() => {
+      email = server.uniqueEmail()
+      return Client.createAndVerify(config.publicUrl, email, password, server.mailbox)
+    })
+
+    it('fails with invalid verification method', () => {
+      return Client.login(config.publicUrl, email, password, {
+        verificationMethod: 'notvalid'
+      })
+        .then(() => {
+          assert.fail('should not have succeed')
+        }, (err) => {
+          assert.equal(err.errno, 107, 'invalid parameter')
+        })
+    })
+
+    it('can force `email` verification', () => {
+      return Client.login(config.publicUrl, email, password, {
+        verificationMethod: 'email'
+      })
+        .then((res) => {
+          client = res
+          assert.equal(res.verificationMethod, 'email', 'sets correct verification method')
+          return client.emailStatus()
+        })
+        .then((status) => {
+          assert.equal(status.verified, false, 'account is not verified')
+          assert.equal(status.emailVerified, true, 'email is verified')
+          assert.equal(status.sessionVerified, false, 'session is not verified')
+          return server.mailbox.waitForEmail(email)
+        })
+        .then((emailData) => {
+          assert.equal(emailData.headers['x-template-name'], 'verifyLoginEmail', 'confirm sign-in link sent')
+          const code = emailData.headers['x-verify-code']
+          assert.ok(code, 'code is sent')
+          return client.verifyEmail(code)
+        })
+        .then((res) => {
+          assert.ok(res, 'verified successful response')
+          return client.emailStatus()
+        })
+        .then((status) => {
+          assert.equal(status.verified, true, 'account is verified')
+          assert.equal(status.emailVerified, true, 'email is verified')
+          assert.equal(status.sessionVerified, true, 'session is verified')
+        })
+    })
+
+    it('can force `email-2fa` verification', () => {
+      return Client.login(config.publicUrl, email, password, {
+        verificationMethod: 'email-2fa'
+      })
+        .then((res) => {
+          client = res
+          assert.equal(res.verificationMethod, 'email-2fa', 'sets correct verification method')
+          return client.emailStatus()
+        })
+        .then((status) => {
+          assert.equal(status.verified, false, 'account is not verified')
+          assert.equal(status.emailVerified, true, 'email is verified')
+          assert.equal(status.sessionVerified, false, 'session is not verified')
+        })
+    })
```

Comment:
Should this also assert that the expected email was sent, like the previous test does?